app/models/organization.rb

```diff
@@ -50,4 +51,8 @@ def districts
   def discardable?
     facility_groups.none? && users.none? && appointments.none?
   end
+
+  def syncable_patients
```

Comment:
🪓 

If we are syncing all patients in an Org we are gonna have a bad time. 😁

Reply:
Addressed in 8fdfb24.